docs/integrations/sources/github.md

```diff
@@ -76,15 +75,17 @@ The Github connector should not run into Github API limitations under normal usa
 ### Requirements
 
 * Github Account;
-* `access_token` - Github Personal Access Token wih the necessary permissions \(described below\);
-* `start_date` - start date for 3 incremental streams: `comments`, `commits` and `issues`;
-* `repository` - Space-delimited list of GitHub repositories/organizations which looks like `<owner>/<repo> <organization>/* <organization_new>/* <owner_new>/<repo_new>`.
-
-**Note**: if you want to specify the organization to receive data from all its repositories, then you should specify it according to the following pattern: `<organization>/*`
+* **Authentication** - Select from 2 authentication methods:
+    * **Authenticate via GitHub (OAuth)** - Authentication method which better suits for Airbyte Cloud, click [Authenticate your account] button to get `access_token`;
+    * **Authenticate with Personal Access Token** - Use this method for Airbyte Open-Source. Log into GitHub and then generate a [personal access token](https://github.com/settings/tokens). To load balance your API quota consumption across multiple API tokens, input multiple tokens separated with `,`;
+* **Start Date** - The date from which you'd like to replicate data for streams: `comments`, `commit_comment_reactions`, `commit_comments`, `commits`, `deployments`, `events`, `issue_comment_reactions`, `issue_events`, `issue_milestones`, `issue_reactions`, `issues`, `project_cards`, `project_columns`, `projects`, `pull_request_comment_reactions`, `pull_requests`, `pull_requeststats`, `releases`, `review_comments`, `reviews`, `stargazers`;
```

Comment:
"start_date" only supported by 17 streams which in docs.

this streams: 
```
assignees
branches
collaborators
issue_labels
organizations
pull_request_commits
pull_request_stats
repositories
tags
teams
users
```

DOES NOT SUPPORT START DATE 
the main reason because this streams does not have `date` key